docs/docs/channels/email/infobip.md

```diff
@@ -0,0 +1,31 @@
+---
+sidebar_position: 13
+---
+
+# Infobip
+
+You can use the [Infobip](https://www.infobip.com/developers/) provider to send transactional emails to your customers using the Novu Platform with a single API to create multi-channel experiences.
+
+## Getting Started
+
+To use the Infobip channel, you will need to create a Infobip account and add your API key and Base URL to the Infobip integration on the Novu platform.
```

Comment:
```suggestion
To use the Infobip provider, you will need to create a Infobip account and add your API key and Base URL to the Infobip integration on the Novu platform.
```

Reply:
This is the case for most of the documentation. Do I go ahead and change it in all the files?

Reply:
Yeah, you can do it in a separate PR. I updated also my suggestion.

Reply:
will do 👍🏽 . Updated